stdlib/Markdown/docs/src/index.md

```diff
@@ -364,12 +364,21 @@ They can be defined using the following `!!!` syntax:
     This warning admonition has a custom title: `"Beware!"`.
 ```
 
-The type of the admonition can be any word, but some types produce special styling,
-namely (in order of decreasing severity): `danger`, `warning`, `info`/`note`, and `tip`.
```

Comment:
I think the purpose of the `/` was to show that `info` and `note` has the same severity.